Source/GUI/GUISkin.h

```diff
@@ -11,10 +11,6 @@ namespace RTE {
 		/// @param Screen GUIScreen Interface.
 		explicit GUISkin(GUIScreen* Screen);
 
-		/// Destructor method used to free a GUISkin object in system
-		/// memory.
-		~GUISkin();
```

Comment:
Isn't this going to cause leaked memory? Or is destroy called somewhere else externally?

I think it'd be more consistent in that case if we make Destroy private instead and just let RAII clean it up in the destructor, instead of requiring the caller to manually Destroy() the skin.